package.json

```diff
@@ -17,7 +18,8 @@
     }
   },
   "scripts": {
-    "test": "./node_modules/.bin/standard && ./node_modules/.bin/mocha test"
```

Comment:
Ok, cool -- done. I can see a command like 'npm run lint-test' more appropriate with the actual execution of logic. IMO, 'npm run test' should let you test a specific file or all files in the test/ path...